mirage-http-CHANGES.md

```diff
@@ -0,0 +1,73 @@
+### 3.1.0
```

Comment:
The file that is picked up is the first one mentioned in `change_logs` [here](http://erratique.ch/software/topkg/doc/Topkg.Pkg.html#VALdescribe). You should be able to specify an alternate one the cli in the `log` and `publish` options via the `--change-log` argument.